python/src/llvm.cc

```diff
@@ -139,8 +139,6 @@ std::string translateLLVMIRToASM(llvm::Module &module,
   {
     llvm::raw_string_ostream stream(result);
     llvm::buffer_ostream pstream(stream);
-    for (llvm::Function &f : module.functions())
-      f.addFnAttr(llvm::Attribute::AlwaysInline);
```

Comment:
Removed this, because this attribute insertion already done on lines 106-108.
Seems that this duplication happened when Hopper support was merged.